src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -212,11 +216,11 @@ public static class PublishedContentExtensions
                 if (cacheAttr != null)
                 {
                     var ctx = new DittoCacheContext(cacheAttr, content, type, culture);
-                    return cacheAttr.GetCacheItem(ctx, () => ConvertContent(content, type, culture, instance, processorContexts, onConverting, onConverted));
+                    return cacheAttr.GetCacheItem(ctx, () => ConvertContent(content, type, umbracoApplicationContextAccessor, culture, instance, processorContexts, onConverting, onConverted));
                 }
                 else
                 {
-                    return ConvertContent(content, type, culture, instance, processorContexts, onConverting, onConverted);
+                    return ConvertContent(content, type, umbracoApplicationContextAccessor, culture, instance, processorContexts, onConverting, onConverted);
```

Comment:
@leekelleher I wanted to find the highest up point in the chain where we used `UmbracoContext.Current` (where the CurrentCulture is being used) then use then pass the same context through to the extension methods to the place where we used to inject it into the processor attributes.

It didn't seem right to new the class in both cases but I see how passing this through might seem a little odd too. :)